### PR TITLE
Fix tests after removal of atv2 See: #8181

### PR DESCRIPTION
**Pull request by razzeee**
Ping @Memphiz

### CODE REVIEW

File: xbmc/utils/test/TestSystemInfo.cpp [hunk 1]
```diff
@@ -228,9 +227,7 @@ TEST_F(TestSystemInfo, GetUserAgent)
   EXPECT_NE(std::string::npos, g_sysinfo.GetUserAgent().find("Windows")) << "'GetUserAgent()' must contain 'Windows'";
 #elif defined(TARGET_DARWIN_IOS)
   EXPECT_NE(std::string::npos, g_sysinfo.GetUserAgent().find("like Mac OS X")) << "'GetUserAgent()' must contain ' like Mac OS X'";
-#ifdef TARGET_DARWIN_IOS_ATV2
-  EXPECT_NE(std::string::npos, g_sysinfo.GetUserAgent().find("CPU OS ")) << "'GetUserAgent()' must contain 'CPU OS '";
-#else  // ! TARGET_DARWIN_IOS_ATV2
+#ifndef TARGET_DARWIN_IOS_ATV2
```
Comment (Memphiz) [inline xbmc/utils/test/TestSystemInfo.cpp]: TARGET_DARWIN_IOS_ATV2 is gone 100% (or at least should be). No need to guard it here.

**Comment by Memphiz** [comment]
Thx for noticing (sry that i didn't ...) - as commented TARGET_DARWIN_IOS_ATV2 should not be used at all anymore.

**Comment by Razzeee** [comment]
Not a problem, I enjoy this kind of work for some odd reason.
Please have another look.

**Comment by Memphiz** [comment]
Fine

**Pull request merged**